librz/core/cprint.c

```diff
@@ -32,6 +32,26 @@ RZ_API RZ_OWN char *rz_core_print_string_c_cpp(RzCore *core) {
 	return rz_strbuf_drain(sb);
 }
 
+/**
+ * \brief Print a colored help option with dynamic alignment.
+ * \param option The option string to print.
+ * \param arg The argument string to print.
+ * \param description The description string to print.
+ * \param maxOptionAndArgLength The maximum combined length of any option + arg strings.
+ */
+RZ_API void rz_core_print_colored_help_option(const char *option, const char *arg, const char *description, size_t maxOptionAndArgLength) {
```

Comment:
As this one doesn't use `RzCore`, lets move it to the `librz/util/print.c` and `librz/include/rz_util/rz_print.h`. Then your changes to include `rz_core.h` in some files would be unnecessary.

Reply:
If you move, rename it as well: `rz_print_colored_help_option()`